spmd/tensor/ops/pointwise_ops.py

```diff
@@ -416,3 +419,28 @@ def pointwise_rule(
 
 for op in pointwise_ops:
     DTensor._op_to_rules[op] = pointwise_rule
+
+
+@register_prop_rule("aten._softmax_backward_data.default")
+def softmax_bwd_rule(op_schema: OpSchema) -> OutputSharding:
+    input_specs = cast(List[DTensorSpec], op_schema.args_spec)
+    ops_dim_map = pytree.tree_map(lambda spec: spec.dim_map, input_specs)
```

Comment:
I think we better not using pytree in sharding rules unless it's absolutely needed. There're only two tensor arguments, we better just directly call `dim_map` instead of using pytree